WISE-Community/WISE
# Added subject filter to OER library authoring

To test:
- Log in as teacher and open a unit in the authoring tool.
- Add an Outside Resource component to a step.
- Make sure OER library appears correctly and you can see two rows of resources before having to scroll.
- Make sure subject filter and search options work.
- Make sure selecting and changing selected OER, inputting directly into URL field work as before.

Resolves #2215.
